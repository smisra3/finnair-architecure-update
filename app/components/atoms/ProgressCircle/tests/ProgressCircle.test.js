import 'jest-styled-components';
import { shallow } from 'enzyme';
import { ProgressRingVanilla } from '../ProgressCircle';

describe('ProgressCircle Component', () => {
  let component = '';
  beforeEach(() => {
    component = shallow(<ProgressRingVanilla />);
  });

  test('should match snapshot ', () => {
    expect(component).toMatchSnapshot();
  });
});
