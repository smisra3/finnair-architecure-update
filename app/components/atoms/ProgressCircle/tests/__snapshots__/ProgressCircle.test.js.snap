// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressCircle Component should match snapshot  1`] = `
<svg
  height={NaN}
  width={NaN}
>
  <circle
    className="underlay"
    fill="transparent"
    r={NaN}
  />
  <circle
    className="progress"
    fill="transparent"
    r={NaN}
    strokeDasharray="NaN NaN"
    style={
      Object {
        "strokeDashoffset": NaN,
      }
    }
  />
  <text
    alignmentBaseline="middle"
    data-slnm-id="undefinedProgressText"
    dy=".1em"
    textAnchor="middle"
    x="50%"
    y="50%"
  />
</svg>
`;
